### PR TITLE
Feature/server/159 endpoint to load already active executions for web

### PR DESCRIPTION
**Pull request by YIllmann**
This PR aims to implement a backend endpoint for loading active executions. To avoid data inconsistency the first step was to persist the runtime test-data into the database.

Two approaches are now possible. The client uses the execution id to identify possible active executions in the run module execution-wise OR the client receives an additionally json containing all the running executions displaying them into an extra div. 

An example FE will be added shortly to state the point.

### CODE REVIEW

**Comment by YIllmann** [comment]
@H1ghBre4k3r @7erra @simohlsen 
I have added a design proposal for the display of active scenarios. Depending on the design the implementation of the backend may vary. As mentioned in the PR description the design 1a/1b supports the latter approach of receiving an extra json of all active executions. The design 2 supports the mapping approach on the client.
IMO the design 1a/b keeps the comunication between client and server very basics and only adds additional fronten implementation, whilst the second design keeps the frontend basic but makes the "client backend" more complicated.
Please state ur opinion :)

**Comment by H1ghBre4k3r** [comment]
I'd prefer approach 1a/b. For me, there should be a clearer separation between templates and running executions.

**Review by H1ghBre4k3r — COMMENTED** [review]
I'd prefer approach 1a/b. For me, there should be a clearer separation between templates and running executions.

File: server/vars.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import os
 
 ACQUIRE_TIMEOUT = 3  # The timeout used to time out a locking-request
+DELIMITER = ","  # delimiter to parse csv strings
```
Comment (simohlsen) [inline server/vars.py:4]: Maybe rename it to `RESULT_DELIMITER` as different uses cases may require other delimiters.

**Comment by YIllmann** [comment]
> I'd prefer approach 1a/b. For me, there should be a clearer separation between templates and running executions.

Based on a democratic approach we have 3v1 for 1a/b
Therefore i will add an implementation for retrieving currently active executions into a separate get request.

**Review by H1ghBre4k3r — APPROVED** [review]

**Review by simohlsen — APPROVED** [review]

**Pull request merged**